osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/EMonthlyBillingValuesPeriod1DSMR4.java

```diff
@@ -22,7 +24,7 @@
 import org.opensmartgridplatform.simulator.protocol.dlms.cosem.processing.UInteger32DataProcessor;
 
 @CosemClass(id = 7)
-public class DataOfBillingPeriod1 extends ProfileGeneric {
+public class EMonthlyBillingValuesPeriod1DSMR4 extends ProfileGeneric {
```

Comment:
The name starting with `E` might be confusing, because this profile also contains mbus values.

Reply:
Corrected this